setup.cfg

```diff
@@ -1,2 +1,64 @@
-[bdist_wheel]
-universal=1
+[metadata]
+name = docrepr
+version = attr: docrepr.__version__
+description = Renders Python docstrings to rich HTML
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/spyder-ide/docrepr
+author = Tim Dumol and the Spyder Development Team
+author_email = spyder.python@gmail.com
+maintainer = Spyder Development Team and Docrepr Contributors
+maintainer_email = spyder.python@gmail.com
+license = BSD-3-clause
+license_file = LICENSE.txt
+license_files =
+    AUTHORS.md
+    LICENSE.txt
+classifiers =
+    Development Status :: 4 - Beta
+    Environment :: Web Environment
+    Framework :: IPython
+    Framework :: Jupyter
+    Framework :: Matplotlib
+    Framework :: Sphinx
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Topic :: Documentation :: Sphinx
+    Topic :: Scientific/Engineering :: Visualization
+    Topic :: Software Development :: Documentation
+    Topic :: Software Development :: Libraries :: Python Modules
+    Topic :: Text Processing :: Markup :: reStructuredText
+    Topic :: Utilities
+keywords = Sphinx docstrings renderer HTML Spyder IPython help formatter
+project_urls =
+    Github = https://github.com/spyder-ide/docrepr
+    Bug Tracker = https://github.com/spyder-ide/docrepr/issues
+    Parent Project = https://www.spyder-ide.org/
+
+[options]
+packages = find:
```

Comment:
Does it exclude the new `tests` directory?

Reply:
The tests are included with the package, per previous requests here, so no. Typically, I use a `src` layout and tests outside the package, but the opposite is the general de-facto convention for Spyder projects, so I went with that here.

Reply:
Seems like this is okay for now.